### PR TITLE
Add cephfs csi doc

### PR DESCRIPTION
**Pull request by Madhu-1**
**Description of your changes:**
updated ceph-csi-drivers.md document to test cephfs csi driver.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)

[skip ci]

### CODE REVIEW

**Comment by Madhu-1** [comment]
@travisn @rootfs PTAL

**Comment by Madhu-1** [comment]
@travisn done PTAL.

File: Documentation/ceph-csi-drivers.md [hunk 1]
```diff
@@ -6,18 +6,22 @@ indent: true
...
 
-Here is a guide on how to use Rook to deploy ceph-csi drivers on a Kubernetes cluster.
+Here is a guide on how to use Rook to deploy ceph-csi drivers on a Kubernetes
+cluster.
 
-- [Enable CSI drivers](#enable-csi-drivers)
-- [Test the CSI driver](#test-the-csi-driver)
-- [Snapshots](#snapshots)
-- [Cleanup](#cleanup)
+- [Enable CSI drivers](#csi-drivers-enabelement)
```
Comment (travisn) [inline Documentation/ceph-csi-drivers.md]: this link has a typo?

File: Documentation/ceph-csi-drivers.md [hunk 2]
```diff
@@ -187,52 +204,78 @@ pvc-c20495c0d5de11e8
 
 # Additional features
```
Comment (travisn) [inline Documentation/ceph-csi-drivers.md]: Should this title be level 2?

File: Documentation/ceph-csi-drivers.md [hunk 3]
```diff
@@ -83,30 +88,37 @@ statefulset.apps/csi-rbdplugin-attacher         1         1         1d
 statefulset.apps/csi-rbdplugin-provisioner      1         1         1d
 ```
 
-## Test the CSI driver
-
 Once the plugin is successfully deployed, test it by running the following example.
 
-### Create the StorageClass
+## Test RBD CSI Driver
```
Comment (travisn) [inline Documentation/ceph-csi-drivers.md]: Should the main titles for RBD and CephFS be level 1 headers?

File: Documentation/ceph-csi-drivers.md [hunk 4]
```diff
@@ -83,30 +88,37 @@ statefulset.apps/csi-rbdplugin-attacher         1         1         1d
...
+
+### Create RBD StorageClass
 
-This [storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml) expect a pool named `rbd` in your Ceph cluster. You can create this pool using [rook pool CRD](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md).
+This
+[storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml)
+expect a pool named `rbd` in your Ceph cluster. You can create this pool using
```
Comment (phlogistonjohn) [inline Documentation/ceph-csi-drivers.md]: expect -> expects

File: Documentation/ceph-csi-drivers.md [hunk 5]
```diff
@@ -123,11 +135,13 @@ data:
   #monitors: BASE64-ENCODED-Comma-Delimited-Mons
 ```
 
-Here, you need your Ceph admin/user password encoded in base64. Run `ceph auth ls` in one of your Ceph pod, encode the key of your admin/user and replace `BASE64-ENCODED-PASSWORD` by your encoded key.
+Here, you need your Ceph admin/user password encoded in base64. Run `ceph auth
+ls` in one of your Ceph pod, encode the key of your admin/user and replace
```
Comment (phlogistonjohn) [inline Documentation/ceph-csi-drivers.md]: pod -> pods

File: Documentation/ceph-csi-drivers.md [hunk 6]
```diff
@@ -83,30 +88,37 @@ statefulset.apps/csi-rbdplugin-attacher         1         1         1d
...
+[rook pool
+CRD](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md).
 
 Please update `monitors` to reflect the Ceph monitors.
 
-### Create the Secret
+### Create RBD Secret
 
-Create a Secret that matches that specified in the [storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml).
+Create a Secret that matches that specified in the
```
Comment (phlogistonjohn) [inline Documentation/ceph-csi-drivers.md]: This is a bit difficult to read. I can't figure out what the secret should contain from this sentence alone.
Do you mean "Create a Secret that matches the namespace and name specified in the storageclass"?

File: Documentation/ceph-csi-drivers.md [hunk 7]
```diff
@@ -178,7 +194,8 @@ spec:
        readOnly: false
 ```
 
-When running `rbd list block --pool [yourpool]` in one of your Ceph pod you should see the created PVC:
+When running `rbd list block --pool [yourpool]` in one of your Ceph pod you
```
Comment (phlogistonjohn) [inline Documentation/ceph-csi-drivers.md]: pod -> pods

File: Documentation/ceph-csi-drivers.md [hunk 8]
```diff
@@ -187,52 +204,78 @@ pvc-c20495c0d5de11e8
 
 # Additional features
 
-## Snapshots
+## RBD Snapshots
 
-This example is based on [kubernetes-csi/external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter), with a few tweaks to make it work along ceph-csi RBD plugin. This is a basic example of the kubernetes snapshot feature. For more information and functionalities please refer to the [volume snapshot documentation](https://kubernetes.io/docs/concepts/storage/volume-snapshots/).
+This example is based on
+[kubernetes-csi/external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter),
+with a few tweaks to make it work along ceph-csi RBD plugin. This is a basic
```
Comment (phlogistonjohn) [inline Documentation/ceph-csi-drivers.md]: along -> "along with the" ?

File: Documentation/ceph-csi-drivers.md [hunk 9]
```diff
@@ -271,3 +314,110 @@ kubectl delete -f snapshotclass.yaml
...
+  userKey: BASE64-ENCODED-PASSWORD
+
+  # Required if provisionVolume is set to true
+  adminID: BASE64-ENCODED-USER
+  adminKey: BASE64-ENCODED-PASSWORD
+```
+
+Here, you need your Ceph admin/user ID and password encoded in base64.  Encode
+admin/user ID in base64 format and replace `BASE64-ENCODED-USER` and Run `ceph
+auth ls` in one of your Ceph pod, encode the key of your admin/user and replace
```
Comment (phlogistonjohn) [inline Documentation/ceph-csi-drivers.md]: pod -> pods

File: Documentation/ceph-csi-drivers.md (hunk 6 above)
Comment (Madhu-1) [reply Documentation/ceph-csi-drivers.md]: create a secret that matches the `adminid` and `userid` specified in storage class.

**Comment by Madhu-1** [comment]
@travisn @phlogistonjohn Thanks for the review. addressed review comments PTAL.

**Pull request merged**